index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Porfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="navbar">
+        <div class="mylogo">
+        <a class="mylogo-text" href="https://twitter.com/Mukumbuta8">Emmanuel Simasiku</a>
+        </div>
+        <nav class="toolbar">
+        <div class="menubar1"></div>
+        <div class="menubar2"></div>
+        <div class="menubar3"></div>
+        </nav>
+    </header>
+    <section class="primary-text">
+        <h1>Hey there. I'm Emmanuel</h1>
+        <h2>I'm a Software Developer</h2>
```

Comment:
- [ ] Your text should be wrapped in a `<p>` tag. Please, kindly change it to a `<p>` tag.